src/applications/check-in/components/pages/TravelComplete/TravelClaimSuccessAlert.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import { hasMultipleFacilities } from '../../../utils/appointment';
+
+const TravelClaimSuccessAlert = props => {
+  const { appointments } = props;
+
+  const { t } = useTranslation();
+
+  if (hasMultipleFacilities(appointments)) {
```

Comment:
This placeholder logic should get a @TODO so we remember to update it once we are doing this logic based off of what is stored in context.

Reply:
I like passing appointments into this component, but I added the todo at the import in the index file

Reply:
I don't think we are going to be managing an array of appointments as the thing in context. Just something simple like an array of objects like this:
[
   {
       facility: [string],
       timestamp: [string],
       multiAppts: [bool]
   }
]